### DISCUSSION TITLE
Cannot run command "splash"

### DISCUSSION THREAD

**Discussion by KutluT** [InteractiveComputerGraphics/SPlisHSPlasH#265] [Q&A]
Hi I've installed SPlisHSplasH using pip install without a virtual environment and also followed everything in the installation process word for word except the command 
`pip install -I build/dist/*.whl`
where it gave an error. Therefore I changed it to 
`pip install -I build/dist/pySPlisHSPlasH-2.12.4-cp311-cp311-win_amd64.whl`
and it worked after that, the installation process was over. However when I tried to run `splash` it gave the error

```
splash : The term 'splash' is not recognized as the name of a cmdlet, function, script file, or operab
le program. Check the spelling of the name, or if a path was included, verify that the path is correct
 and try again.
At line:1 char:1
+ splash
+ ~~~~~~
    + CategoryInfo          : ObjectNotFound: (splash:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```
I've never used a program of this sort before and I'm still not much experienced in programming so if the solution is really obvious, sorry for not identifying it. If possibly, I would prefer a quick response since its an urgent job.

**Comment by codylico** [comment]
_outsider comment; feel free to ignore:_
This looks like a PowerShell output. I don't know if PowerShell supports running Python scripts by name this way (assuming `splash` is a Python script).

What is the output of the following command (taken from [here](https://github.com/InteractiveComputerGraphics/SPlisHSPlasH/blob/master/doc/py_getting_started.md))?
```sh
python -c "import pysplishsplash"
```
That should help determine if SPlisHSPlasH was installed correctly.

**Comment by janbender** [comment]
Can you take a look if you find the splash executable in a subfolder (Scripts) of your Python installation?
